src/analysis/AppLauncher.js

```diff
@@ -43,337 +43,320 @@ const ApplicationLauncher = _.flow(
     title: _.get('application'),
     activeTab: appLauncherTabName,
   })
-)(
-  (
-    {
-      name: workspaceName,
-      sparkInterface,
```

Comment:
I am not sure I understand what the fix is to get to the Spark console, what is replacing the sparkInterface here?